src/mixin.js

```diff
@@ -0,0 +1,75 @@
+const MESSAGE = Symbol.for('message');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyObject = require('yamlify-object');
+// eslint-disable-next-line import/no-extraneous-dependencies
+const yamlifyColors = require('yamlify-object-colors');
+
+const utils = require('./utils');
+const Message = require('./message');
+const Colorizer = require('./colorizer');
+const defaultErrorColors = require('./colors');
+
+function ColorizedMixin(instance, options = {}) {
+  options = Object.assign({}, options, {
+    prefix: '',
+    postfix: '',
+    stackTrace: true,
+    colors: defaultErrorColors,
+    types: yamlifyColors,
+  });
+  var oldLog = instance.log.bind(instance);
+  instance.log = function log(info, callback) {
+    var remainingInfo = Object.assign({}, info);
+    delete remainingInfo.from;
+    delete remainingInfo.stack;
+    delete remainingInfo.trace;
+    delete remainingInfo.message;
+    delete remainingInfo.timestamp;
+    delete remainingInfo.label;
```

Comment:
I think `level` needs to be removed, too.

Reply:
Easy to change. Can you tell me what behavior is happening so I can write a test for it?

Reply:
Winston logs made with `logger.info('blah')` render as something like:

    [INFO] blah
      - level: 'info'

Reply:
👍 I totally noticed after commenting and forgot to comment again

Its fixed in the new formatter, and the test is updated